src/H5Fmpi.c

```diff
@@ -81,6 +82,10 @@
  * Programmer:  Quincey Koziol
  *              Friday, January 30, 2004
  *
+ * Programmer:  Jonathan Kim
```

Comment:
I'm guessing this is from syncing with develop, but we should probably remove this section, especially since it conflicts with the original programmer section (that I'd remove too).

Reply:
I removed Jonathan's credit. We can talk later about removing programming credits throughout the library in general.